packages/client-twitter/src/utils.ts

```diff
@@ -170,7 +168,10 @@ export async function sendTweet(
     twitterUsername: string,
     inReplyTo: string
 ): Promise<Memory[]> {
-    const tweetChunks = splitTweetContent(content.text);
+    const tweetChunks = splitTweetContent(
+        content.text,
+        Number(client.runtime.getSetting("MAX_TWEET_LENGTH"))
```

Comment:
what happens if this is not set? will passing a falsish value into `splitTweetContent` cause a problm?